### ISSUE TITLE
NetBeans Does not send data to wakatime dashboard anymore

### ISSUE THREAD

**Issue by doblador42** [wakatime/netbeans-wakatime#24]
The plugin seems to not be working with netbeans version 12.4

**Comment by doblador42** [comment]
Here is what the netbeans log about wakatime
 
```
WARNING [org.netbeans.modules.autoupdate.updateprovider.AutoupdateCatalogParser]: https://raw.githubusercontent.com/wakatime/netbeans-wakatime/master/updates.xml:1:49: Document root element "module_updates", must match DOCTYPE root "null".
WARNING [org.netbeans.modules.autoupdate.updateprovider.AutoupdateCatalogParser]: https://raw.githubusercontent.com/wakatime/netbeans-wakatime/master/updates.xml:1:49: Document is invalid: no grammar found.
```

**Comment by franciscofl12** [comment]
Same for me here, is not working and I thought that it was because I didn't configure it well.

**Comment by alanhamlett** [comment]
Fixed in v4.0.11 just released.
